moped-database/views/project_list_view.sql

```diff
@@ -72,12 +75,6 @@ AS WITH project_person_list_lookup AS (
         ) as project_feature,
     fsl.funding_source_name,
     ptl.type_name,
-    ( -- get the most recent status_update (project note type 2)
-      SELECT mpn.project_note
-      FROM moped_proj_notes mpn
-        WHERE mpn.project_id = mp.project_id AND mpn.project_note_type = 2 AND mpn.is_deleted = false
-        ORDER BY mpn.date_created DESC
-        LIMIT 1) AS project_note,
```

Comment:
oh nice! I see where `project_note` used to come from. 👍 So cool to see something fresh in this view with the lateral join. 🚀